docs/api.rst

```diff
@@ -38,8 +38,14 @@ Models
    .. function:: strcmp
 
 EVM
----
+====
```

Comment:
EVM should still be under the API reference level, not on its own level

![image](https://user-images.githubusercontent.com/2467355/33391343-af89e62a-d506-11e7-8ebb-1b5b4ac0e86a.png)